frontend/src/app/main/ui/workspace/textpalette.cljs

```diff
@@ -129,13 +129,19 @@
      [:span.left-arrow {:on-click on-left-arrow-click} i/arrow-slide]
 
      [:div.color-palette-content {:ref container :on-wheel on-wheel}
-      [:div.color-palette-inside
-       (for [[idx item] (map-indexed vector current-typographies)]
-         [:& typography-item
-          {:key idx
-           :file-id file-id
-           :selected-ids selected-ids
-           :typography item}])]]
+     (if (empty? current-typographies)
+        [:div.color-palette-empty {:style {:position "absolute"
+                                           :left "50%"
+                                           :top "50%"
+                                           :transform "translate(-50%, -50%)"}} 
+              "There are no Typography Styles in your Library yet"]
```

Comment:
This should be translated, with something similar that what it do in line 139 of this file.

You should use the tr function with a key. That key should exist in frontend/translations/en.po

Reply:
Hi @superalex, this is my first time dealing with translations on this community and  I was reading about the contributing guidelines for translations on the penpot community. It got me a little confused as to what should my approach be in this scenario. Should I directly edit the frontend/translations/en.po and add the key or should that be done via Weblate.
In continuation to this, should I add translations to all languages that exist on the platform.

Reply:
Sorry @akshay-gupta7 I think we must update the doc about that part to make it easier.

Editing directly frontend/translations/en.po adding the new key and the translation is enough, the rest of the languages will be translated in the future via Weblate.

Thank you very much!

Reply:
Thank you so much for the clarification @superalex. I shall resolve this accordingly.